Pod/Classes/ios/NYTPhotosOverlayView.m

```diff
@@ -49,6 +50,10 @@ - (void)layoutSubviews {
     }];
     
     [super layoutSubviews];
+
+    if ([self.captionView respondsToSelector:@selector(setPreferredMaxLayoutWidth:)]) {
```

Comment:
> `conformsToProtocol:` requires a global runtime lock; `respondsToSelector:` does not

Interesting. Where's this documented? I don't see if in the docs for `NSObject`.